src/Gridify.Elasticsearch/GridifyExtensions.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using Elastic.Clients.Elasticsearch;
+using Elastic.Clients.Elasticsearch.QueryDsl;
+using Gridify.Syntax;
+
+namespace Gridify.Elasticsearch;
+
+public static class GridifyExtensions
+{
+   public static Query ToElasticsearchQuery<T>(this string? filter, IGridifyMapper<T>? mapper = null)
+   {
+      if (string.IsNullOrWhiteSpace(filter))
+         return new MatchAllQuery();
+
+      var syntaxTree = SyntaxTree.Parse(filter, GridifyGlobalConfiguration.CustomOperators.Operators);
```

Comment:
There are some small warnings that if you already have null checks like this example, it is easy to fix using a `!` operator. 
``` csharp
var syntaxTree = SyntaxTree.Parse(filter, GridifyGlobalConfiguration.CustomOperators.Operators);
```

Github analyzer already marked these so I'll skip other ones.